osbs/tekton.py

```diff
@@ -592,6 +599,11 @@ def has_succeeded(self):
         return status_reason == 'Succeeded'
 
     def has_not_finished(self):
+        data = self.data
```

Comment:
I notice we do this assignment in code more often. Do we need this or could simply use self.data?

Reply:
self.data is property which calls get_info to get fresh object

IF we would be using self.data everywhere, it would be constantly getting new data even tho they are still the same

I guess it isn't required in all cases (when used just once), but it certainly doesn't hurt

Reply:
Yes, I was refering to cases when it is used just once - such as this one. But it is not a big issue.